templates/definition/meter/huawei-sun2000.yaml

```diff
@@ -146,11 +146,11 @@ render: |
     timeout: 30s
     reset: 1 # reset watchdog on normal
     set:
-      source: switch
+      source: sequence
```

Comment:
erst switch, dann sequence. das muss geschachtelt sein!

Reply:
Danke, hatte noch mehr Fehler in der Syntax. Hab dies soeben angepasst…